Probesys/bileto
# imp: Allow to select observers during tickets creation

## Related issue(s)



https://github.com/Probesys/bileto/issues/852

## How to test manually



1. Click on "New ticket" with a user who can edit actors of a ticket
2. Select one or several observers and submit the ticket
3. Verify that the observers are set on the ticket
4. Edit the observers
5. Verify that it still works

## Checklist
